src/utils.ts

```diff
@@ -186,19 +186,14 @@ export const checkOptions = ({
   prefixes: Record<string, string>;
   appsWhiteList: string[] | null | undefined;
 }): void => {
-  if (!latitude || !longitude) {
+  if ((!latitude && !longitude) || !address) {
```

Comment:
suggestion: I think this logic might still not be exactly right - this will currently trigger the exception for every case where lat/lng + address aren't both supplied. I think it should be this:

```suggestion
  if (!(latitude && longitude) && !address) {
```